Make getSpace return an optional LayoutUnit

rwlbuis opened the pull request:
#### 9b4310720574078092e96301115af54867be0956
<pre>
Make getSpace return an optional LayoutUnit
<a href="https://bugs.webkit.org/show_bug.cgi?id=242188">https://bugs.webkit.org/show_bug.cgi?id=242188</a>

Reviewed by Darin Adler.

Make getSpace return an optional LayoutUnit.

* Source/WebCore/rendering/RenderBoxModelObject.cpp:
(WebCore::getSpace):
(WebCore::RenderBoxModelObject::calculateBackgroundImageGeometry const):

Canonical link: <a href="https://commits.webkit.org/252067@main">https://commits.webkit.org/252067@main</a>
</pre>

darinadler reviewed (approved):
I would have said "instead of using -1" in the bug title.

rwlbuis committed:
Make getSpace return an optional LayoutUnit
https://bugs.webkit.org/show_bug.cgi?id=242188

Reviewed by Darin Adler.

Make getSpace return an optional LayoutUnit.

* Source/WebCore/rendering/RenderBoxModelObject.cpp:
(WebCore::getSpace):
(WebCore::RenderBoxModelObject::calculateBackgroundImageGeometry const):

Canonical link: https://commits.webkit.org/252067@main
